widgetssdk/src/main/java/com/glia/widgets/core/engagement/GliaEngagementRepository.java

```diff
@@ -60,6 +64,12 @@ public boolean hasOngoingEngagement() {
         return gliaCore.getCurrentEngagement().isPresent();
     }
 
+    public boolean isCallVisualizerEngagement() {
```

Comment:
Maybe use IsCallVisualizerUseCase?

Reply:
@Pelkar I'm not sure. In this case we would need to pass UseCase to Repository class in two places. However, I don't see any Repository that has UseCase inside it. So it will stand out overall.

I agree that having two ways to define if it's call visualizer engagement is not good. So, I came up with the [following improvement](https://github.com/salemove/android-sdk-widgets/pull/496/commits/341d78e0a72cd7193ea8c26f3188b1fb5ae9ed52). Does it make sense to you or you have a better idea?

Reply:
Yes, this solution looks good 👍